Dockerfile

```diff
@@ -5,17 +5,17 @@ ARG https_proxy
 ARG local_conan_server
 
 # Replace the default profile and remotes with the ones from our Ubuntu build node
-ADD "https://raw.githubusercontent.com/ess-dmsc/docker-ubuntu18.04-build-node/master/files/default_profile" "/root/.conan/profiles/default"
+ADD "https://raw.githubusercontent.com/ess-dmsc/docker-ubuntu18.04-build-node/7a4840422e78d451ada97662399116c31a4bc44e/files/default_profile" "/root/.conan/profiles/default"
```

Comment:
I am not sure that we want to make these changes to the *Dockerfile*. On the other hand seeing as we have replaced this file in the *v3.0.0* branch, we could just let it be for now. Any comments @matthew-d-jones ?

Reply:
It seems that these changes were made by @matthew-d-jones so I will close this comment as resolved.